website/servers/index.php

```diff
@@ -45,13 +46,25 @@
 	if (strpos($server, '.') === false) {
 		die("invalid server location");
 	}
+	$email = isset($_POST['email']) ? trim($_POST['email']) : false;
+	if (!$email || !strpos($email, '@')) {
```

Comment:
```suggestion
	if (!$email || strpos($email, '@') === false) {
```

I understand that yours also handles the `0` case, but writing it this way is much more readable.